linux_os/guide/system/auditing/policy_rules/audit_rules_for_ospp/bash/shared.sh

```diff
@@ -0,0 +1,310 @@
+# platform = Red Hat Enterprise Linux 7,Red Hat Enterprise Linux 8
+
+# let's start with a clean slate
+rm -f /etc/audit/rules.d/*
```

Comment:
We do not preserve any customization this way, but that's fine by me. It would be unnecessarily complex otherwise.

Reply:
Remediation doesn't remove everything now. There is no strict need for this. Other audit rules can co-exist with OSPP audit rules.